Cpp/source/DrawingEngine/DrawingEngine.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <iostream>
+#include <stdint.h>
+struct Pixel
+{
+    uint8_t R {};
+    uint8_t G {};
+    uint8_t B {};
+};
+class DrawingEngine
+{
+protected:
+    int buffer_size;
+    Pixel** FrameBuffer;
```

Comment:
It would be nice if FrameBuffer was its own class, so drawing engine does not have to worry about allocating and freeing memory

Reply:
done